app/services/courses/creation_service.rb

```diff
@@ -37,6 +37,9 @@ def build_new_course
         course.program_type = 'teacher_degree_apprenticeship'
         course.can_sponsor_student_visa = false
         course.can_sponsor_skilled_worker_visa = false
+        course.additional_degree_subject_requirements = nil
+        course.degree_subject_requirements = nil
```

Comment:
Do we actually need line 40 and 41? They are nil by default

Reply:
True! I amended that :).